emp/emp-yamls/values.yaml

```diff
@@ -0,0 +1,24 @@
+# A sample values.yaml file for users who want to install EMP related components on EKS cluster manually, using helm chart.
+images: # images used by EMP components
+  eks_vol_watcher:
+  emp_pod_webhook:
+  evm_autoscaler:
+  evm_vpc_cni:
+  evm_vpc_cni_init:
+manualInstallation: 'true' # (*)
```

Comment:
Do we need single quotes for the boolean value?

Reply:
ah no, my bad